src/ProgramBlockEditor.test.js

```diff
@@ -180,6 +191,105 @@ test('When a step is clicked, action panel should render next to the step', () =
     }
 });
 
+
+describe("Expand button and add nodes", () => {
```

Comment:
I think "Expand add nodes toggle switch and add nodes"

Reply:
Done.